gap_linux.go

```diff
@@ -53,6 +53,10 @@ func (a *Advertisement) Configure(options AdvertisementOptions) error {
 	for _, uuid := range options.ServiceUUIDs {
 		serviceUUIDs = append(serviceUUIDs, uuid.String())
 	}
+	var serviceData = make(map[string][]byte)
+	for uuid, data := range options.ServiceData {
+		serviceData[New16BitUUID(uuid).String()] = data.([]byte)
+	}
```

Comment:
My guess would be that you need to use `map[string]interface{}` instead of `map[string][]byte` (like manufacturer data but with a different key type).